src/base/io/utilities/convertOldStyleModel.m

```diff
@@ -0,0 +1,109 @@
+function model = convertOldStyleModel(model)
```

Comment:
I guess it is important as it ensures backward compatibility ... You can also rename the fields temporarily from an already 'new' model, and then test whether they get changed. 

We can also put this off for later.